v03_pipeline/lib/misc/io.py

```diff
@@ -17,6 +20,30 @@ def does_file_exist(path: str) -> bool:
     return os.path.exists(path)
 
 
+def file_size_bytes(path: str) -> int:
```

Comment:
I asked [zulip](https://hail.zulipchat.com/#narrow/stream/123010-Hail-Query-0.2E2-support/topic/Possible.20to.20access.20the.20size.20of.20Hail.20Table.2FMatrixTable.3F) if there was a better way, and I don't think there is 😢